server/src-lib/Hasura/GraphQL/Execute/LiveQuery/Poll.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE CPP #-}
```

Comment:
None of those changes seem related to PDV. Do we need them? Is this just a bad merge?